airflow-core/src/airflow/cli/commands/connection_command.py

```diff
@@ -375,7 +375,8 @@ def connections_test(args) -> None:
         raise SystemExit(1)
 
     print("\nTesting...")
-    status, message = conn.test_connection()
+    # TODO: Revisit this
+    status, message = conn.test_connection()  # type: ignore
```

Comment:
Will this work or fail currently -- with this change?

Reply:
Reverted the change, i dont think it makes sense to use sdk Connection here tbh. WDYT?